kylin-spark-project/kylin-spark-engine/src/main/scala/org/apache/kylin/engine/spark/job/CubeBuildJob.java

```diff
@@ -329,7 +329,7 @@ private void build(Collection<NBuildSourceInfo> buildSourceInfos, SegmentInfo se
             Preconditions.checkState(!toBuildCuboids.isEmpty(), "To be built cuboids is empty.");
             Dataset<Row> parentDS = info.getParentDS();
 
-            if (toBuildCuboids.size() > 1) {
+            if (toBuildCuboids.size() >= 1) {
```

Comment:
'toBuildCuboids' is not empty, I think we can remove this judgment.

Reply:
good suggestion ~